distribution/src/bin/elasticsearch-env

```diff
@@ -35,6 +35,31 @@ ES_HOME=`dirname "$ES_HOME"`
 # now set the classpath
 ES_CLASSPATH="$ES_HOME/lib/*"
 
+
+check_glibc_version() {
+  # We need to check the version of glibc, because JDK >=15 requires
+  # glibc 2.14
+  GLIBC_VERSION="$( getconf GNU_LIBC_VERSION 2>/dev/null | cut -d' ' -f2)"
+  if [[ $? -ne 0 ]]; then
+    # Must be on a POSIX system that isn't using glibc.
+    return 0
+  fi
+
+  if echo "$GLIBC_VERSION" | awk '{ if ($1 >= 2.14) { exit 1 } else { exit 0 } }'; then
```

Comment:
Is this doing a numeric comparison? Is it casting $1 to a float? Are there any floating point precision issues possible?